packages/terra-card/src/Card.scss

```diff
@@ -3,8 +3,15 @@
     background-color: var(--terra-card-background-color, #fff);
     border: var(--terra-card-border, 1px solid #c8cacb);
     border-radius: var(--terra-card-border-radius, 3px);
-    box-shadow: var(--terra-card-box-shadow, none);
     color: var(--terra-card-color, inherit);
     overflow: hidden;
   }
+
+  .default {
+    box-shadow: var(--terra-card-box-shadow, none);
+  }
+
+  .raised {
+    box-shadow: var(--terra-card-raised-box-shadow, none);
```

Comment:
@neilpfeiffer let me know if you have a style you want applied by default for this class in terra-core. Otherwise, it's just a new themeable option we can utilize

Reply:
@alex-bezek - Yes, we will want to have a visual version for default theme, otherwise it would look like a dead prop (not recommended). I'll grab a box-shadow style to use.

Reply:
@alex-bezek Values to used for default terra-opensource-theme:
```
background: white;
border: 1px solid #c8cacb; 
box-shadow: 0 3px 8px -2px rgba(0, 0, 0, 0.18), 0 3px 10px 0 rgba(0, 0, 0, 0.12), 0 3px 3px -3px rgba(0, 0, 0, 0.3);
```